code/modules/research/ordnance/scipaper_partner.dm

```diff
@@ -4,21 +4,20 @@
 	accepted_experiments = list(/datum/experiment/ordnance/explosive/lowyieldbomb)
 	multipliers = list(SCIPAPER_COOPERATION_INDEX = 0.75, SCIPAPER_FUNDING_INDEX = 0.75)
 	boostable_nodes = list(
-		"bluespace_basic" = 2000,
-		"NVGtech" = 1500,
-		"practical_bluespace" = 2500,
-		"basic_plasma" = 2000,
-		"basic_mining" = 2000,
-		"adv_mining" = 2000,
+		"bluespace_theory" = 2000,
+		"night_vision" = 1500,
+		"anomaly_research" = 2500,
+		"mining" = 2000,
+		"mining_adv" = 2000,
```

Comment:
honestly stuff like this is gonna keep happening unless we tie these string keys to `#define`s. There should be a science-based file in the `__DEFINES` directory, and we should just start tying those to defines

```dm
#define BLUESPACE_THEORY_NODE "bluespace_theory"
```

Then, we would replace that here in `boostable_nodes` and wherever we equivalency check it at in the code (probably on the datum for bluespace theory). It's a bit more work, but this issue will literally never happen again in this manner if we take the time to do it right. This is because if a define doesn't exist, the code will fail to compile, forcing the coder to fix it. That means if someone changes the define to `THEOREM_OF_BLUESPACE_NODE` for whatever flippant reason, the code will break and they will be forced to fix it since CI will not pass.